public/app/features/integrated-alerting/components/AlertRules/AddAlertRuleModal/AddAlertRulesModal.constants.ts

```diff
@@ -20,3 +20,6 @@ export const SEVERITY_OPTIONS = [
     label: Messages.severities.SEVERITY_WARNING,
   },
 ];
+
+// Seconds
```

Comment:
Do we need to know that those are seconds? :)

Reply:
yeah, maybe units don't really matter here